content/en-us/reference/engine/classes/Accoutrement.yaml

```diff
@@ -5,23 +5,52 @@ memory_category: Instances
 summary: |
   An object that can attach to a player's character.
 description: |
-  An Accoutrement welds its child `Class.Part|part` called "Handle" to the
+  An Accoutrement welds its child `Class.BasePart|part` called "Handle" to the
   player's character. You can change the position and rotation of the Handle
-  part using the `Class.Accoutrement.AttachmentPos|AttachmentPos`,
-  `Class.Accoutrement.AttachmentRight|Right`,
-  `Class.Accoutrement.AttachmentForward|Forward`, and
-  `Class.Accoutrement.AttachmentUp|Up` properties.
+  part using either the `Class.Accoutrement.AttachmentPoint` property or
+  an `Class.Attachment` within the Handle.
+
+  When an accoutrement is attached to a character it will attach to the Head by default.
+  A `Class.Weld` named "HeadWeld" will be created in the head with its `Class.Weld.Part0|Part0`
+  set to the Head and its `Class.Weld.Part1|Part1` set to the Handle. Its `Class.Weld.C0|C0`
+  will be set to `0, 0.5, 0` and its `Class.Weld.C1|C1` will be set to the Accoutrement's.
+  `Class.Accoutrement.AttachmentPoint`. The `Class.Weld.C1` will update automatically when the
+  `Class.Accoutrement.AttachmentPoint` changes. The HeadWeld will automatically
+  be destroyed if the Accoutrement is removed from the character.
```

Comment:
```suggestion
  When an `Class.Accoutrement` is attached to a character model, it will attach to the "Head" part by default.
  A `Class.Weld` named "HeadWeld" will be created in the head with its `Class.Weld.Part0|Part0`
  set to the Head and its `Class.Weld.Part1|Part1` set to the Handle. Its `Class.Weld.C0|C0`
  will be set to `0, 0.5, 0` and its `Class.Weld.C1|C1` will be set to the Accoutrement's.
  `Class.Accoutrement.AttachmentPoint`. The `Class.Weld.C1` will update automatically when the
  `Class.Accoutrement.AttachmentPoint` changes. The HeadWeld will automatically
  be destroyed if the Accoutrement is removed from the character.
```